agent/config/config_test.go

```diff
@@ -69,6 +70,9 @@ func TestAgent_Merge(t *testing.T) {
 			TLSServerName: "cows-or-pets",
 			SkipVerify:    true,
 		},
+		Policy: &Policy{
+			DefaultCooldown: 1800000000000,
```

Comment:
feels like something that should be a constant or a var somewhere since these values are repeated so much :)